my_pyscf/mcscf/lassi_states.py

```diff
@@ -158,6 +158,13 @@ def all_single_excitations (las):
         len (all_states) - len (ref_states), len (ref_states)))
     return las.state_average (weights=weights, charges=charges, spins=spins, smults=smults)
 
+def get_chargetrans_excitations(las,CT=1):
```

Comment:
Can we not do this right now? This whole method is so incredibly experimental that I don't want to get stuck with functions that don't make sense 6 months down the line. Its name is also not strictly accurate (starting at CT=2, states that aren't charge transfer but spin shuffling can appear)